nano/node/wallet.hpp

```diff
@@ -185,6 +185,10 @@ class wallet_representative_counts
 public:
 	uint64_t voting{ 0 }; // Representatives with at least the configured minimum voting weight
 	uint64_t half_principal{ 0 }; // Representatives with at least 50% of principal representative requirements
+	bool have_half_rep () const
```

Comment:
This code seems to not be used

Reply:
It was used when I looked through earlier, got dropped in the conflict resolution merge it seems.